libs/blocks/quiz/quiz.css

```diff
@@ -1,980 +1,384 @@
-.quiz { 
-  min-height: 1000px;
-  display: flex;
-  align-items: center;
-  flex-direction: column;
+.quiz-page * {
+  box-sizing: border-box;
 }
 
-.quiz > div:not([class]) {
-  display:none;
+.quiz-page .feds-topnav-wrapper {
+  padding: 0 20px;
```

Comment:
This was a hold over from the old stuff that I persisted. It's there because without it, our customized nav has the logo and sign-in hugging the edge uncomfortably. If you think this should be removed because it could interfere with gnav in some way, I can certainly do it